app/src/main/java/com/keylesspalace/tusky/EditProfileActivity.kt

```diff
@@ -113,6 +114,7 @@ class EditProfileActivity : BaseActivity(), Injectable {
             accountFieldEditAdapter.addField()
             if(accountFieldEditAdapter.itemCount >= MAX_ACCOUNT_FIELDS) {
                 it.isEnabled = false
+                it.isVisible = false
```

Comment:
Please delete the `it.isEnabled = false`, its not needed when the button is invisible anyway